core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -96,20 +97,39 @@ private[spark] class SparkSubmit extends Logging {
   }
 
   /**
-   * Kill an existing submission using the REST protocol. Standalone and Mesos cluster mode only.
+   * Kill an existing submission.
+   * Standalone, Kubernetes and Mesos cluster mode only.
```

Comment:
Since this is pluggable, I don't think this comment makes sense anymore.

Reply:
Ok will update.